### PR TITLE
Apollo client setup. Implemented Countries Api.

### PR DESCRIPTION
**Pull request by Strishwork**

### CODE REVIEW

File: app/src/main/java/com/example/countryinfo/Apollo.kt [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    check(Looper.myLooper() == Looper.getMainLooper()) {
+        "Only the main thread can get the apolloClient instance"
+    }
+
+    if (instance != null) {
+        return instance!!
+    }
+
+    instance = ApolloClient.builder()
+        .serverUrl("https://countries-274616.ew.r.appspot.com/")
```
Comment (yeoliinyk) [inline app/src/main/java/com/example/countryinfo/Apollo.kt]: Let's move it to constant. Companion object would be fine for now.

File: app/src/main/graphql/com/example/countries.graphql [hunk 2]
```diff
@@ -0,0 +1,54 @@
+query getCountries{
```
Comment (yeoliinyk) [inline app/src/main/graphql/com/example/countries.graphql]: Let's add a space before all opening braces.

**Review by yeoliinyk — APPROVED** [review]

File: app/src/main/java/com/example/countryinfo/Apollo.kt (hunk 1 above)
Comment (Strishwork) [reply app/src/main/java/com/example/countryinfo/Apollo.kt]: Done

File: app/src/main/graphql/com/example/countries.graphql (hunk 2 above)
Comment (Strishwork) [reply app/src/main/graphql/com/example/countries.graphql]: Changed

**Pull request merged**